docs/adapting-new-services.md

```diff
@@ -178,8 +177,17 @@ steps:
         operations: ['concatenate']
 ```
 
-## 7. Docker Container Images
+There are default limits set by the environment variables `MAX_BATCH_INPUTS` and
+`MAX_BATCH_SIZE_IN_BYTES`. Providers should consult the `env-defaults.ts` file to obtain the
```

Comment:
I think if you provide a relative link to env-defaults.ts, GitHub will do the right thing and link to it, which may make it ever so slightly easier to figure out.

Reply:
Done.